fastlane/jetpack_metadata/default/subtitle.txt

```diff
@@ -1 +1 @@
-Faster, safer WordPress
+Build your WordPress website & blog; create posts, add photos & track analytics!
```

Comment:
Unfortunately, App Store Connect will reject this because the subtitle has to be 30 characters at most.

![image](https://user-images.githubusercontent.com/1218433/180089155-418011a0-b0ff-410a-96ab-931e0ceb7e6d.png)

I'll leave a comment about this in the internal request, too.

Reply:
Thanks. I updated this to say "Supercharge your WordPress".